test/token/ERC721/ERC721.behavior.js

```diff
@@ -359,20 +368,20 @@ function shouldBehaveLikeERC721 (
 
         context('to a receiver contract returning unexpected value', function () {
```

Comment:
This test is failing because you've changed the test code in a way that the behavior will be different.